pyanaconda/ui/gui/xkl_wrapper.py

```diff
@@ -62,26 +62,11 @@ def __init__(self):
         self._rxkb = rxkb.Context()
 
         self._layout_infos = {}
-        self._build_layout_infos()
+        _build_layout_infos(self._rxkb)
 
         self._switch_opt_infos = {}
         self._build_switch_opt_infos()
 
-    def _build_layout_infos(self):
-        for layout in self._rxkb.layouts.values():
-            name = layout.name
-            if layout.variant:
-                name += ' (' + layout.variant + ')'
-
-            langs = []
-            for lang in layout.iso639_codes:
-                if iso639.find(iso639_2=lang):
-                    langs.append(iso639.to_name(lang))
-
-            if name not in self._layout_infos:
-                self._layout_infos[name] = LayoutInfo(langs, layout.description)
-            else:
-                self._layout_infos[name].langs.extend(langs)
```

Comment:
I would squash this commit with the first one so it is visible that this method was moved not removed.